code/global.js

```diff
@@ -3,16 +3,20 @@
  */
 
 var gist = {
-	clientId: 'da931d37076424f332ef',
+	// clientId: 'da931d37076424f332ef',
+	clientId: 'Ov23liPosltrsNbxULJG',
```

Comment:
We need this for testing purposes. The original `clientId` will be used afterwards.